File: client/src/services/newCourseServices.js
import axios from 'axios'

export const SendPDFFileForCourse = async(pdfFile, courseId)=>{
  
  const config = {
    headers : {
      'Content-Type': 'multipart/form-data'
    }
  };

  return await axios.post(`${process.env.REACT_APP_API_URL}/course/${courseId}/add-pdf`,pdfFile,config);
}

export const GetAnswer = async (data) => {
  return await axios.get(`${process.env.REACT_APP_API_URL}/course/${data.courseId}/answer?question=${data.question}`);
};

File: main_service/services/new_course_services.py
import requests
from decouple import config

PYTHON_SERVICE_API_URI = config('PYTHON_SERVICE_API_URI')

def upload_new_pdf_and_send_to_service(pdf_file):
    files = {
        'file' : pdf_file
    }

    requests.post(f'{PYTHON_SERVICE_API_URI}/course/test_name', files=files)